plugins/nanospell/plugin.js

```diff
@@ -48,6 +48,7 @@
 		SPELLCHECK_COMPLETE: 'spellCheckComplete',
 		SPELLCHECK_ABORT: 'spellCheckAbort'
 	};
+	var BLOCK_REQUEST_LIMIT = 5;
```

Comment:
We can actually probably put this into the plugin settings:  https://github.com/PolicyStat/ckeditor-spell-check-plugin-js-dev-edge/blob/b562b640c816679d4ac404228ee9fcf0fbd317e3/plugins/nanospell/plugin.js#L191

and then do something like `BLOCK_REQUEST_LIMIT = editor.config.nanospell.blockRequestLimit || DEFAULT_BLOCK_REQUEST_LIMIT`